src/main/java/edu/byu/cs/honorChecker/HonorCheckerCompiler.java

```diff
@@ -20,15 +21,15 @@ public class HonorCheckerCompiler {
      * @return the path to the .zip file
      */
     public static String compileSection(int section) {
-        int sectionID = CanvasIntegration.sectionIDs.get(section);
+        int sectionID = CanvasIntegrationImpl.sectionIDs.get(section);
```

Comment:
Why are you using the `CanvasIntegrationImpl` here instead of some getter on `CanvasIntegration.getCanvasIntegration()`?

Reply:
Will we want to keep this hardcoding of section id's permanently? I thought this was temporary so I didn't bother taking it further

Reply:
Hmm, I see. I was thinking that they would ultimately still come from our `CanvasIntegration`, but more likely they are going to come from the database.

Good point!

Reply:
I might be able to get something to ask canvas for section id's, in which case it could go in the interface

Reply:
#262

Reply:
You made quick work of that 👀 

I haven't run it yet, but I like what I saw from my cursory review